<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarrageAssistantDlg</class>
 <widget class="QDialog" name="BarrageAssistantDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>689</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>689</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BarrageAssistantDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_main{
background:rgba(0,0,0,1);
}
#wgt_title,#wgt_content{
border:1px solid #999999;
border-top:none;
border-left:none;
border-right:none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#ffffff;
}
QComboBox{
border:1px solid #999999;
border-radius:2px;
background:transparent;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#ffffff;
}
QComboBox::drop-down {
   width:7px;
  border:none;
  padding-right:4px;
  padding-left:0px;
}
QComboBox::down-arrow {
image: url(:/TQLive/images/arrow.png);
}

QComboBox QAbstractItemView  
{  
     border: 1px solid #999999;  
     outline:0px;
    font-size:12px;
}  
  
QComboBox QAbstractItemView::item  
{  
    height: 20px;  
}  
  
QComboBox QAbstractItemView::item:selected  
{     
    background-color: lightgray;  
}
QListView::item{
background:gray;
    font-size:12px;
}

#btn_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_normal.png);
background-repeat:no-repeat;
border:none;
}
#btn_close:hover{
background-image: url(:/TQLive/images/pb_close_pressed.png);
}
#btn_close:pressed{
margin:1px;
padding:1px;
}


</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>11</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="text">
            <string>弹幕助手</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>39</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="lbl_fontNo">
             <property name="text">
              <string>字号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combox_fontNo">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_close">
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>13</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>12</width>
               <height>13</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>11</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_content" native="true">
        <property name="styleSheet">
         <string notr="true">#lbl_tip{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#ffffff;
padding-left:6px;
}
#textEdit_content{
background:transparent;
border:none;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_tip">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>58</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>58</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style='line-height:25%'&gt;欢迎使用探球弹幕助手&lt;/p&gt;&lt;p  style='line-height:25%'&gt;左键拖动窗口位置，移动边缘可以拉伸改变大小&lt;/p&gt;&lt;p  style='line-height:25%'&gt;右键调出设置菜单&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_content">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_gift" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>270</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>270</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#textEdit_gift{
background:transparent;
border:none;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit_gift">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
